configure.ac

```diff
@@ -15,7 +15,7 @@ GNOME_COMPILE_WARNINGS(maximum)
 
 AC_PATH_PROG([GLIB_GENMARSHAL], [glib-genmarshal])
 
-PYTHON=python2
+PYTHON=python3
```

Comment:
It specifically installs for python3 
We need toolkit to be 2&3 compatible

Reply:
yes, I know. I don't yet have a fix for installing for both Python 2 and Python 3.

I am leaving this change as is for now, since LEM doesn't hold here.
Reference: https://en.wikipedia.org/wiki/Law_of_excluded_middle

Reply:
@pro-panda so the sugar toolkit is not yet completely 2 & 3 compatible?

Reply:
It is compatible.  You can override PYTHON when you call configure.ac, right?

Reply:
Yes @quozl , as per my undersatnding in that case it will install either python 2 or python 3 and it totally depends upon user how they want to install the toolkit.